### ISSUE TITLE
Wrong center locations in BandPassActivityProfile

### ISSUE THREAD

**Issue by pcostanza** [broadinstitute/gatk#5765]
### Affected tool(s) or class(es)
org.broadinstitute.hellbender.utils.activityprofile.BandPassActivityProfile

### Affected version(s)
Latest master branch as of March 6, 2019

### Description 
Line 147 should read:
```Java
final SimpleInterval loc = getLocForOffset(superState.getLoc(), i)
```
instead of
```Java
final SimpleInterval loc = getLocForOffset(justAddedState.getLoc(), i)
```

Likewise, line 154 should read:
```Java
states.add(superState)
```
instead of
```Java
states.add(justAddedState)
```

Otherwise, the location changes in org.broadinstitute.hellbender.utils.activityprofile.ActivityProfile, line 241 doesn't make a lot of sense, and neither does the extension of `getMaxProbPropagationDistance()` in org.broadinstitute.hellbender.utils.activityprofile.BandPassActivityProfile, lines 100-102.

**Comment by davidbenjamin** [comment]
@pcostanza You're correct; the current code does not respect the propagation of the case `ActivityProfileState.Type.HIGH_QUALITY_SOFT_CLIPS`.
